examples/connext_dds/CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ set(CMAKE_MODULE_PATH
 )
 
 find_package(RTIConnextDDS
-    "6.0.0"
+    "6.1.0"
```

Comment:
This example only works with 6.1.0. This is why I changed the requirements for RTIConnextDDS